src/bioregistry/data/bioregistry.json

```diff
@@ -95353,7 +95353,7 @@
     "homepage": "https://ensayosclinicos-repec.ins.gob.pe/en/",
     "logo": "https://ensayosclinicos-repec.ins.gob.pe/images/plantilla/img_logo_principal_repec_en.png",
     "name": "Peruvian Clinical Trial Registry",
-    "pattern": "^\\d+-\\d+$",
+    "pattern": "^\\d+-\\d+$|^\\d+-\\d+-[A-Z]$",
```

Comment:
This pattern I think could be simplified by starting with the original pattern and adding the -[A-Z] part as optional at the end.